styleguide/component.stories.js

```diff
@@ -9,7 +9,6 @@ import initTables from '../src/js/tables';
 import GreedyNav from '../src/ts/greedy-nav';
 
 // eslint-disable-next-line no-unused-vars
-import locals from './haml_locals.rb'; // just used to watch
```

Comment:
Watching this file has caused intermittent build errors and after adding the footer links I found it 100% of the time caused build errors.

Will need to find a better way to watch this, and other dependant files (it is possible to extend the loader to do this).